doc/install_pcl_from_source.md

```diff
@@ -0,0 +1,37 @@
+# Install PCL from source
+
+## Install PCL from source without SSE
+
+This is needed for [AttentionClipper](https://github.com/jsk-ros-pkg/jsk_recognition/issues/2380).
```

Comment:
I prefer `This is sometimes needed for AttentionClipper`, because this installation is not always required (and perhaps we won't need this procedure on default environment in the future).

Reply:
thx, I updated.